benchmark/benchmark.py

```diff
@@ -1,33 +1,62 @@
+#!/usr/bin/env python
 '''
 Created on 04.10.2019
 
 @author: gocht
 '''
+import argparse
 import sys
 import benchmark_helper
 import pickle
 
+# Available tests
 tests = ["bm_baseline.py", "bm_simplefunc.py"]
 
+# Available instrumenters
 instrumenters = ["profile", "trace", "dummy", "None"]
 if sys.version_info.major >= 3:
     instrumenters.extend(["cProfile", "cTrace"])
 
-# How many times the instrumented code is run during 1 test run
+# Default values for: How many times the instrumented code is run during 1 test run
 reps_x = {
     "bm_baseline.py": ["1000000", "2000000", "3000000", "4000000", "5000000"],
     "bm_simplefunc.py": ["100000", "200000", "300000", "400000", "500000"],
 }
-# How many times a test invocation is repeated (number of timings per test instance)
-test_repetitions = 51
 
-bench = benchmark_helper.BenchmarkEnv(repetitions=test_repetitions)
+
+def str_to_int(s):
+    try:
+        return int(s)
+    except ValueError:
+        return int(float(s))
+
```

Comment:
Please make that an int(float(s)) in all cases. It's more readable and the uncommon case where we use more than 53 bits can be neglected imo.